src/components/Accounts/editProfilePassword.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { getAuth, updatePassword } from 'firebase/auth'
+import { hideHomeSearch, setUser } from '../../redux/actions'
+import Footer from '../Other/footer'
+
+export default function EditProfilePassword () {
+  const dispatch = useDispatch()
+  const [newPassword, setNewPassword] = useState('')
+  const [newConfirmPassword, setNewConfirmPassword] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+  const user = useSelector(state => state.user)
+
+  const auth = getAuth()
```

Comment:
small nit - this line was at the top of the function in the other files, would be good to keep them consistent. also, we usually group together all state variables separate from hooks like `useDispatch` which go at the top.